res.qrc

```diff
@@ -43,6 +43,7 @@
         <file>img/taskbar/light/taskbar_event.svg</file>
         <file>img/transfer.svg</file>
         <file>translations/bg.qm</file>
+	<file>translations/cs.qm</file>
```

Comment:
>@zetok Please let contributors fix their bugs before merging. Helps to avoid some nasty merge conflicts in the future.

Noted.
Sorry, looks like I've been a little bit too quick to merge - though, given how long qTox was stalled, merging PRs that I could, even if they would have some minor issues that wouldn't affect quality of qTox build, seemed like a good way to go about things.